libarchive/archive_write_add_filter_b64encode.c

```diff
@@ -146,12 +146,17 @@ static int
 archive_filter_b64encode_open(struct archive_write_filter *f)
 {
 	struct private_b64encode *state = (struct private_b64encode *)f->data;
-	size_t bs = 65536, bpb;
+	ssize_t bpb;
+	size_t bs = 65536;
```

Comment:
Actually, `get_bytes_per_block` is defined to return an `int`.   So `bpb` here should be `int` to match that.  That means that `bs` should probably also be `int`.